taskgraph/Task.py

```diff
@@ -1336,22 +1348,29 @@ def join(self, timeout=None):
         return successful_wait
 
     def get(self, timeout=None):
-        """Return the result of the `func` once it is ready.
+        """Return the result of the ``func`` once it is ready.
 
-        If `timeout` is None, this call blocks until the task is complete
-        determined by a call to `.join()`. Otherwise will wait up to `timeout`
-        seconds before raising a `RuntimeError` if exceeded.
+        If ``timeout`` is None, this call blocks until the task is complete
+        determined by a call to ``.join()``. Otherwise will wait up to
+        ``timeout`` seconds before raising a `RuntimeError` if exceeded.
 
         Args:
             timeout (float): if not None this parameter is a floating point
                 number specifying a timeout for the operation in seconds.
 
         Returns:
             value of the result
+
         Raises:
-            RuntimeError when `timeout` exceeded.
+            RuntimeError when ``timeout`` exceeded.
+            ValueError if ``store_result`` was set to ``False`` when the task
+                was created.
 
         """
+        if not self._store_result:
+            raise ValueError(
+                'must set `store_result` to True in `add_task` to invoke this '

```

Comment:
missing double tick marks around `add_task`. I see you were cleaning up some of these.

Reply:
Sorry, never mind. I see this is just a string output for the exception.